api/src/schema/index.js

```diff
@@ -125,6 +125,11 @@ const Schema = i18n => {
       evaluations: [Evaluation]
     }
 
+    type ErsRating @cacheControl(maxAge: 90) {
+      measurement: Int
+      upgrade: Int
+    }
+
```

Comment:
Do we know that these are always integers? I used floats for the previous values because I was looking at numbers like `12.0`.

Reply:
As a side note, it's a shame that `measurement` and `upgrade` return data in a different format than `walls`, which also has `measurement` and `upgrade` keys. Nothing we can do, I guess, but it might surprise consumers of the API down the road.

Reply:
oh man you're totally right, good call, ty!

Also a good flag, I'll ask David about that